core/server/services/webhooks/trigger.js

```diff
@@ -0,0 +1,63 @@
+const _ = require('lodash');
+const common = require('../../lib/common');
+const models = require('../../models');
+const pipeline = require('../../../server/lib/promise/pipeline');
+const request = require('../../../server/lib/request');
+const debug = require('ghost-ignition').debug('services:webhooks:trigger');
+
+function makeRequests(webhooksCollection, payload, options) {
+    _.each(webhooksCollection.models, (webhook) => {
+        const event = webhook.get('event');
+        const targetUrl = webhook.get('target_url');
+        const webhookId = webhook.get('id');
+        const reqPayload = JSON.stringify(payload);
+
+        models.Webhook.edit({'last_triggered_at': Date.now()}, {id: webhookId}).catch((err) => {
+            debug('Unable to update webhooks last_triggered_at in db', err);
+            return Promise.reject(err);
```

Comment:
Is there a best practice on how to handle errors like these? 🤔

Reply:
the webhook update here happens in background, no one will listen on the reject.

Reply:
Yes! 🙊 Do we just leave it with `common.logging.warn` error message here?

Reply:
yeah just logging 👍